packages/E2ETest/wdio/pages/BasePage.ts

```diff
@@ -3,89 +3,44 @@
  * Licensed under the MIT License.
  */
 
-import {
-  REACT_CONTROL_ERROR_TEST_ID,
-  BACK_BUTTON,
-  TREE_DUMP_RESULT,
-} from '@react-native-windows/tester/js/examples-win/LegacyTests/Consts';
+import { TREE_DUMP_RESULT } from '@react-native-windows/tester/js/examples-win/LegacyTests/Consts';
+
+const ELEMENT_LOADED_TIMEOUT = 60000;
 
 export function By(testId: string): WebdriverIO.Element {
   return $('~' + testId);
 }
 
-export function wait(timeout: number) {
-  return new Promise(resolve => {
-    setTimeout(resolve, timeout);
-  });
-}
-
 export class BasePage {
   isElementLoaded(element: string): boolean {
     return By(element).isDisplayed();
   }
 
-  waitForElementLoaded(element: string, timeout?: number) {
+  waitForElementLoaded(element: string) {
     browser.waitUntil(
-      () => {
-        return this.isElementLoadedOrLoadBundleError(element);
-      },
-      this.timeoutForPageLoaded(timeout),
-      'Wait for element ' + element + ' timeout'
+      () => this.isElementLoaded(element),
+      ELEMENT_LOADED_TIMEOUT,
+      `Failed to find element with testId "${element}" within ${ELEMENT_LOADED_TIMEOUT}ms`
     );
   }
 
-  getTreeDumpResult(): boolean {
-    var testResult = false;
-    const maxWait = 20;
-    var waitCount = 1;
-    do {
-      testResult = this.treeDumpResult.getText() === 'TreeDump:Passed';
-      if (!testResult) {
-        console.log(
-          '####Waiting for treedump comparison result ' +
-            waitCount +
-            '/' +
-            maxWait +
-            '...####'
-        );
-        // #6041 this line doesn't do anything because of missing await, but
-        // adding the await causes test failures.
-        // eslint-disable-next-line @typescript-eslint/no-floating-promises
-        wait(100);
-        waitCount += 1;
-      }
-    } while (waitCount <= maxWait && !testResult);
-
-    return testResult;
-  }
-
-  protected timeoutForPageLoaded(currentTimeout?: number) {
-    if (currentTimeout) {
-      return currentTimeout;
-    }
-    return this.waitforPageTimeout;
-  }
-
-  protected get homeButton() {
-    return By(BACK_BUTTON);
-  }
-
-  private get reactControlErrorMessage() {
-    return By(REACT_CONTROL_ERROR_TEST_ID);
```

Comment:
Nothing with the test ID this points to seems to currently exist in either RN or RNW. Removed the check for this, since it was just adding log spam.